### ISSUE TITLE
Where to find the RL training section while running the code in example directory

### ISSUE THREAD

**Issue by Xyc-arch** [huawei-noah/SMARTS#994]
**High Level Description**
[find the RL training section while running the code in example directory]

**Desired SMARTS version**
[0.4.12]

**Operating System**
[ubuntu 20.04]

**Problems**
[I wanna design the RL training part of the simulation in example directory. However I cannot find it? Could anyone tell me which directory to look for?]

**Comment by Adaickalavan** [comment]
Examples on
+ how to use SMARTS environment, and 
+ how to train agents in SMARTS environment using reinforcement learning (rl) algorithms

are given in the `/SMARTS/examples` folder.

For example, see 
+ `/SMARTS/examples/driving_in_traffic`
    + https://github.com/huawei-noah/SMARTS/tree/example-driving-in-traffic/examples/driving_in_traffic
    + Trains an agent to drive as fast and as far as possible, without colliding or going off-road, in heavy traffic using DreamerV2.
+   `/SMARTS/examples/rllib`
    + https://github.com/huawei-noah/SMARTS/tree/develop/examples/rllib
    + Trains agents using RLlib library.
+ `/SMARTS/examples/ray_multi_instance`
    + https://github.com/huawei-noah/SMARTS/blob/develop/examples/ray_multi_instance.py
    + Trains agents using (i) Ray to distribute training and evaluation, and (ii) PyTorch to implement the agent policy.
+  `/SMARTS/baselines/marl_benchmark`
    + https://github.com/huawei-noah/SMARTS/tree/develop/baselines/marl_benchmark
    + This directory contains the scenarios, training environment, and agents used in the CoRL20 paper.

Additionally, to speed up data collection, you may consider using the provided `ParallelEnv` wrapper to simulate multiple SMARTS environments in parallel. Usage of `ParallelEnv` is illustrated in
+ `/SMARTS/examples/parallel_environment.py` 
+ https://github.com/huawei-noah/SMARTS/blob/develop/examples/parallel_environment.py

We hope the above addresses your question and thus this issue is being closed. Please feel free to reopen this issue if your problem persists or open a new issue for other questions.
